app/src/store/state.ts

```diff
@@ -79,6 +79,7 @@ export const stateModel: StateModelIF = {
     getNameReservationFailed: false,
     helpMeChooseModalVisible: false,
     affiliationErrorModalValue: NrAffiliationErrors.NONE, // initially hidden
+    isLearBusiness: false,
```

Comment:
Does this need to get reset if the user aborts the current name request flow and starts anew?

Reply:
It is calculated each time a new business is entered